### DISCUSSION TITLE
Problem with [resolve #issuenumber] action while push

### DISCUSSION THREAD

**Discussion by kyrylchenko** [community/community#65345] [Actions]
### Select Topic Area

Bug

### Body

I have project board with todos issues  refered and created for different repos but showed together in one project board.
count of issue number starts for each repo personaly from 1.
the problem is if i have 3 repos(repoA,repoB,repoC) issues with same number #3
and when i do commit in repoA with **`resolve #3`** it automaticly closing all repo's issue which i have in my project board with number #3

**Comment by geoffreylgv** [comment]
Hello @kyrylchenko, 

Some approaches of solutions I can say.
- Try this:
 `closes basURI/repoName#issueNumber`
 `resolves barURI/repoName#issueNumber`
 - Or try this:
 `closes repoName#issueNumber`
 `resolves repoName#issueNumber`
 
 **baseURI**: is the githubusername (if the repository is on your own, then, it will be your GH username), or the organization GH username
 **repoName**: is the repository name
**issueNumber**: is the number of your issue you want to close

**NB**: _never use multi repo for one project board, so try, and let us know your thoughts. ([closes or close || resolves or resolve] use the keyword to close or resolve as usual)_
